posthog_test.go

```diff
@@ -302,6 +305,7 @@ func TestEnqueue(t *testing.T) {
 					"version":     "1.0.0",
 					"platform":    "macos", // :)
 				},
+				SendFeatureFlags: false,
```

Comment:
Can we have a default `false` so people who don't care about this don't have to insert it?

Reply:
Unfortunately go doesn't support default values :( . However this won't impact the end user because omitting a sendfeatureflags property won't trigger that codepath that injects feature flags.